orekit/tests/validate_events.py

```diff
@@ -82,6 +91,9 @@
     epoch0_poliastro,
 )
 
+point = GeodeticPoint(47 * DEG_TO_RAD, 45 * DEG_TO_RAD, 5.0)  # lat, lon, h
```

Comment:
I just saw the latitude value here is `47 deg` but `46` is specified in the poliastro `SatelliteVisibilityEvent`.

Reply:
Thanks, @jorgepiloto, for pointing that out! The cause of the mismatch of times is still puzzling...I will try to take a closer look at this PR again sometime this month.